docs/versions

```diff
@@ -1,3 +1,3 @@
-master              master
+main              main
```

Comment:
this will break existing links, we should look into redirecting

Reply:
Can you explain a bit? Shouldn't links still work as we just renamed the default branch?

Reply:
https://docs.cosmos.network/master/ the url here would change from master to main, it may invalidate urls and links in other repos

Reply:
https://github.com/cosmos/cosmos-sdk/blob/main/docs/.vuepress/enhanceApp.js#L2